lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1442,6 +1475,10 @@ protected function getInsertColumnList()
             }
 
             if (! $this->class->isIdGeneratorIdentity() || $this->class->identifier[0] !== $name) {
+                if (! $this->class->isInsertable($name)) {
```

Comment:
Can you use `$this->class->fieldMappings[$name]['notInsertable']` here please? property access is so much faster than dynamic methods.